torch/_dynamo/trace_rules.py

```diff
@@ -127,6 +128,24 @@
 
 
 """
+
+_TLS = threading.local()
+
+
+@contextmanager
+def dont_wrap_top_module():
+    old = getattr(_TLS, "wrap_top_module", True)
+    _TLS.wrap_top_module = False
+    try:
+        yield False
+    finally:
+        _TLS.wrap_top_module = old
+
+
+def should_wrap_top_module():
+    return getattr(_TLS, "wrap_top_module", True)
```

Comment:
is the plan to remove this once export/compiled autograd tests are fixed?

Reply:
Yes. Would like your help with compiled autograd. 

Export requires some decent amount of work.